plugins/inputs/postfix/README.md

```diff
@@ -29,9 +29,12 @@ $ sudo chmod g+r /var/spool/postfix/maildrop
 
 Posix ACL:
 ```sh
-$ sudo setfacl -Rdm u:telegraf:rX /var/spool/postfix/{active,hold,incoming,deferred,maildrop}
+$ sudo setfacl -Rdm g:telegraf:rX /var/spool/postfix/{active,hold,incoming,deferred,maildrop}
 ```
 
+Note: The ACL sets permissions on the group instead of the user. This because we want to make sure postdrop group still has -wx permissions.
+      If we would set the ACL to the user, telegraf would be unable to send emails via postdrop.
```

Comment:
Let's remove this explanation, it will be confusing after we make the change to fix the setfacl line.